pinax/stripe/models.py

```diff
@@ -24,8 +24,16 @@ class Meta:
         abstract = True
 
 
+class AccountRelatedStripeObject(StripeObject):
```

Comment:
Would it be better to have two abstract classes instead of the hierarchy (mixin pattern) ? This way only specific models that need the field can use it. Example:

```
class Customer(StripeObject, AccountRelatedStripeObject):
    ...
```
If you think Stripe Connect's business logic touches every model, then it makes sense, but I'm not sure it does.